packages/customize-widgets/src/components/focus-control/index.js

```diff
@@ -52,21 +52,26 @@ export default function FocusControl( { api, sidebarControls, children } ) {
 			focusWidget( widgetId );
 		}
 
+		let previewBound = false;
+
 		function handleReady() {
 			api.previewer.preview.bind(
 				'focus-control-for-setting',
 				handleFocus
 			);
+			previewBound = true;
 		}
 
 		api.previewer.bind( 'ready', handleReady );
 
 		return () => {
 			api.previewer.unbind( 'ready', handleReady );
-			api.previewer.preview.unbind(
-				'focus-control-for-setting',
-				handleFocus
-			);
+			if ( previewBound ) {
```

Comment:
Huh, didn't know that it will error if it's not registered (unlike native `removeEventListener`) 😆

Reply:
It's not that `unbind` would crash when the event handler is not registered, but that the `previewer.preview` object whose `.unbind` method is called doesn't even exist. Because it's created only just before the `ready` event is fired (i.e., when the preview frame is loaded).

Here we subscribe to the `ready` event, and then immediately unsubscribe before `ready` was fired. At that time, `previewer.preview` doesn't exist yet.